### DISCUSSION TITLE
MAX_TEXTURE_IMAGE_UNITS(16)

### DISCUSSION THREAD

**Discussion by m061i6** [mozilla/hubs#5405] [Hubs Cloud Questions]
I got a problem with this
```
three.module.js:18814 THREE.WebGLProgram: Shader Error 0 - VALIDATE_STATUS false
Program Info Log: FRAGMENT shader texture image units count exceeds MAX_TEXTURE_IMAGE_UNITS(16)
```
Does anyone got it before?
How to fix it?
Thanks

![image](https://user-images.githubusercontent.com/13673776/166133155-bc27ca82-f54a-4e81-a92c-f8bd72701075.png)

**Comment by rawnsley** [comment]
I've not seen this problem before. I don't think it can be investigated without access to the room where the problem happens. If you aren't able to share it publicly, you should consider sending it privately to the Hubs team for investigation.
